src/Zope2/Startup/tests/test_schema.py

```diff
@@ -43,26 +42,23 @@ class WSGIStartupTestCase(unittest.TestCase):
 
     def setUp(self):
         self.default_encoding = ZPublisher.HTTPRequest.default_encoding
+        self.TEMPNAME = tempfile.mkdtemp()
+        self.TEMPVAR = os.path.join(self.TEMPNAME, "var")
+        os.mkdir(self.TEMPVAR)
```

Comment:
What about using a context manager when need instead of storing the created directory in an instance variable?